### PR TITLE
BL-722-fe-i-want-to-register-as-an-instructor

### PR DESCRIPTION
**Pull request by Rlauckern**
## Description

Created a step by step instructor registration process. Added form to show education and technical skills of potential new instructors. Cleaned up css and buttons to make sure it demonstrates that you (the parent or instructor) are on the correct page. 
Updated/created:
RegisterStep1.js
InstructorWelcome.js
InstructorFlowStep_2.js
SuccesfulSubmission.js

Fixes # (issue)

https://www.loom.com/share/0a7f53e1b11a41849ea18731b1cd80cf

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [✅] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [✅] My code follows the style guidelines of this project
- [✅] I have performed a self-review of my own code
- [✅I have removed unnecessary comments/console logs from my code
- [✅] I have made corresponding changes to the documentation if necessary (optional)
- [✅] My changes generate no new warnings
- [✅] I have checked my code and corrected any misspellings
- [✅] No duplicate code left within changed files
- [✅] Size of pull request kept to a minimum
- [✅] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by vasilii-garanin — APPROVED** [review]
Hi guys, thanks for showing the new flow. My first impression is wow it looks great! The way you have your code set up it will be easy to hook up once next steps are finalized in back end.

**Review by Irving-Delgad0 — APPROVED** [review]
This looks great! definitely a great addition to the site

**Review by AnthonyLayne — APPROVED** [review]
Looks good, would recommend getting rid of the console.logs in the InstructorFlow_Step2.js, and maybe organizing the imports a little. Great Job!

**Review by gclancy121 — DISMISSED** [review]
Looks good to me, no obvious bugs or errors

**Review by Roadlyfe — DISMISSED** [review]
Looks good! clean code. No obvious bugs.

**Review by vasilii-garanin — DISMISSED** [review]
Your changes look good 👍

**Review by dshipuk — DISMISSED** [review]
Code looks great. Good job!

**Review by DawsonReschke — APPROVED** [review]
Looks good, nice job.

**Review by DawsonReschke — DISMISSED** [review]
Looks good, nice job.

File: src/components/pages/Registration/InstructorRegistration/InstructorFlow_Step2.js [hunk 1]
```diff
@@ -56,33 +71,24 @@ const InstrRegForm = () => {
...
-    // .catch(e => {
-    //   console.error(e);
-    //   setFormWarning(e);
-    // });
-
-    // TODO: remove this stub
-    history.push('/instructor-register-success');
+    // This form will be updated with location data will be sent for review
+    axiosWithAuth(authState.idToken)
+      .post('/', {
```
Comment (paulstgermain) [inline src/components/pages/Registration/InstructorRegistration/InstructorFlow_Step2.js:76]: We don't want to be pushing up code that doesn't work - and this won't work! We'll want to send this instructor application data to the route described in the BE pull request, `instructors/register`. 😀

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This is some really solid implementation here, and I'm excited to get this much closer to creating Okta users from within our app's registration process!

That said, there are a couple of things I'd love to address before we can merge this PR up:

File: src/components/pages/Registration/InstructorRegistration/InstructorFlow_Step2.js (hunk 1 above)
Comment (Rlauckern) [reply src/components/pages/Registration/InstructorRegistration/InstructorFlow_Step2.js]: Ok I believe i just updated it properly! Thank you Paul!

**Review by reneetro — APPROVED** [review]
Looks good! Seems like it will be a lot easier for instructors to navigate the pages!

**Review by Roadlyfe — APPROVED** [review]
Refractor looks good. Clean code.

**Review by Cassius-Cassity — APPROVED** [review]
Excellent work not only fixing errors in the code but also making sure you implemented styling along with it.

**Review by AnthonyLayne — APPROVED** [review]
This looks better, thank you for doing the recommendations I had. Great job!

**Review by reneetro — APPROVED** [review]
Looks great! Naming is very clear and the labels for buttons are also much more concise!

**Review by paulstgermain — APPROVED** [review]

**Review by AlexiusThomas — COMMENTED** [review]
Code looks good, clean and easy to understand

**Pull request merged**